test/image-blueprints/layer2-presubmit/group1/rhel94-source-tuned.toml

```diff
@@ -0,0 +1,106 @@
+name = "rhel-9.4-microshift-source-tuned"
+description = "A RHEL 9.4 image with MicroShift and OLM built from source."
+version = "0.0.1"
+modules = []
+groups = []
+distro = "rhel-94"
+
+[[packages]]
+name = "microshift"
+version = "{{ .Env.SOURCE_VERSION }}"
+
+[[packages]]
+name = "microshift-greenboot"
+version = "{{ .Env.SOURCE_VERSION }}"
+
+[[packages]]
+name = "microshift-networking"
+version = "{{ .Env.SOURCE_VERSION }}"
+
+[[packages]]
+name = "microshift-selinux"
+version = "{{ .Env.SOURCE_VERSION }}"
+
+[[packages]]
+name = "microshift-low-latency"
+version = "{{ .Env.SOURCE_VERSION }}"
+
+[[packages]]
+name = "microshift-test-agent"
+version = "*"
+
+[[packages]]
+name = "systemd-resolved"
+version = "*"
+
+{{- if and (env.Getenv "UNAME_M" "") (eq "x86_64" .Env.UNAME_M) }}
+# Kernel RT is only available for x86_64
+[customizations.kernel]
+name = "kernel-rt"
+{{- end }}
```

Comment:
Should we specify this as an explicit dependency in RPM spec?

Reply:
No, there's no kernel rt for ARM and kernel rt is not required for low latency, so we need to enable it only for x86_64